packages/dvilela/skills/memeooorr_abci/prompts.py

```diff
@@ -24,68 +24,76 @@
 Here's your persona:
 "{persona}"
 
-You come up with funny ideas for new tweets and you post them on Twitter.
+You come up with ideas for new tweets that match your persona and you post them on Twitter (aka X).
+
 Your task it to create a new tweet based on your persona.
 """
 
 ANALYZE_FEEDBACK_PROMPT = """
-You are a cryptocurrency and token expert. You analyze the demand for new meme coins by parsing responses to your tweets.
+You are a cryptocurrency and token expert with a specific persona. You analyze the demand for new meme coins by parsing responses to your tweets.
 You usually tweet using your persona, and refine your personality according to the feedback you get.
-Sometimes, when your tweets get a lot of engagement, you decide to create a meme token based on your persona and invest some ETH on it.
+Sometimes, when your tweets get a lot of engagement, you decide to create a meme token based on your persona and invest some {ticker} on it.
```

Comment:
@dvilelaf you'll have to finish this by adding support for ticker. Otherwise this won't work on Celo